### PR TITLE
To do List 

### PR DESCRIPTION
**Pull request by Have-Samuel**
**To do List Project**
We:

1. Made sure no Linter errors
2. Used HTML, CSS & Javscript best practices
3. Applied the webpack
4. Git & Github Workflows
5. Documented professionally

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<body>
+    <div class="item-container">
+        <div class="item-container__title">
+            <p class="title">To do List</p>
+            <i class="icon-top fa-solid fa-arrows-rotate"></i>
+        </div>
+        <form class="item-container__list-content" id="form-container">
+            <label for="item"></label>
+            <input type="text" name="item" id="item" placeholder="Add to your list..." />
+            <!-- <i class="icon-bin fa-solid fa-trash-can"></i> -->
```
Comment (Deepakdanger) [inline src/index.html]: - [ ] Kindly remove the commented code.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Have-Samuel, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] After adding the to-do in the list, While editing and deleting I am getting this error. Kindly resolve this.
- 6905b319-fbaa-4599-91fa-317ae5d10c54
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Have-Samuel, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] As of now the delete function is working fine, but editing the TO-DO is not working, kindly fix it.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @Have-Samuel,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**